tsconfig.json

```diff
@@ -26,6 +26,8 @@
       "@mui/x-tree-view/*": ["./packages/x-tree-view/src/*"],
       "@mui/x-license-pro": ["./packages/x-license-pro/src"],
       "@mui/x-license-pro/*": ["./packages/x-license-pro/src/*"],
+      "@mui-internal/test-utils": ["./node_modules/@mui/monorepo/packages/test-utils/src"],
+      "@mui-internal/test-utils/*": ["./node_modules/@mui/monorepo/packages/test-utils/src/*"],
```

Comment:
Was this change also necessary? 🤔 
Did you also need to add this on top of the `babel.config.js` entry?

Reply:
If I may add some context from a code infra perspective. We'd like to move away from the babel defined aliases. It's not the right layer in the stack to do aliasing. Unfortunately some of the tooling still relies on it. The ultimate goal though is to have a single source of truth for aliasing, the aim is for this to be the tsconfig file.